src/wallet/rpcwallet.cpp

```diff
@@ -2260,6 +2260,7 @@ UniValue getwalletinfo(const UniValue& params, bool fHelp)
             "  \"keypoololdest\": xxxxxx,    (numeric) the timestamp (seconds since GMT epoch) of the oldest pre-generated key in the key pool\n"
             "  \"keypoolsize\": xxxx,        (numeric) how many new keys are pre-generated\n"
             "  \"unlocked_until\": ttt,      (numeric) the timestamp in seconds since epoch (midnight Jan 1 1970 GMT) that the wallet is unlocked for transfers, or 0 if the wallet is locked\n"
+            "  \"paytxfee\": x.xxxx,         (numeric) the transaction fee configuration, set in btc/kb\n"
```

Comment:
This will be handled when I pull in bitcoin/bitcoin#6504, which will already have many merge conflicts everywhere we've already changed this, so I'm inclined to maybe just leave it for now? OTOH, easy to make a commit and then revert it before pulling.